test_lifecycle.py

```
@@ -0,0 +1,75 @@
+ADMIN_PASSWORD = 'AdminPass123!'
+
+print('='*80)
+print('LIFECYCLE DIAGNOSTIC ENDPOINT TEST')
+print('='*80)
+
+# Step 1: Authenticate
+print('\nStep 1: Authenticating...')
+resp = requests.post(
+    f'{BASE_URL}/auth/login',
+    json={'email': ADMIN_EMAIL, 'password': ADMIN_PASSWORD},
+    verify=False,
```

Ruff (0.14.1)

Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)